src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/IconDetailRow.razor.scss

```
@@ -0,0 +1,26 @@
+    color: var(--bit-clr-fg-sec);
+}
+
+.icon-detail-row-value {
+    margin: 0;
+    min-width: 0;
+    overflow: hidden;
+    font-size: 0.8125rem;
+    line-height: 1.4;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    font-family: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
```

Stylelint (17.14.0)

[error] Expected quotes around "SF Mono" (font-family-name-quotes)

(font-family-name-quotes)